cmd/cosign/cli/verify/verify.go

```diff
@@ -132,7 +133,12 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 			return errors.Wrap(err, "initializing piv token verifier")
 		}
 	case certRef != "":
-		pubKey, err = loadCertFromFileOrURL(c.CertRef)
+		certs, err := loadCertFromFileOrURL(c.CertRef)
+		if err != nil {
+			return err
+		}
+		cert := certs[0]
```

Comment:
In loadCertFromFileOrURL, should we enforce that only one cert was provided in the reference? Alternatively, outside of the scope of this PR, should we support attempting verification with multiple trusted certs/keys?

Reply:
i think for now we only use one cert, so we could enforce that. i can make that change.

ref https://github.com/sigstore/cosign/issues/950 for discussion about verification w/ multiple keys!